pkg/apis/configuration/v1/kongupstreampolicy_types.go

```diff
@@ -0,0 +1,117 @@
+package v1
+
+// REVIEW: do we want to create a separate package for Gateway API extensions?
```

Comment:
I'm not sure if we'd like to add a new package but I'd say this should land in `v1` right at the start. `v1beta1` or `v1alpha1` would make much more sense to me.

Reply:
Moved to `v1beta1`. 👍